godotproject/enemy-Ant.gd

```diff
@@ -0,0 +1,89 @@
+extends KinematicBody2D
+
+signal fireball_start
```

Comment:
add documentation to game design doc for signal

Reply:
signal is uneeded to re-arch on enemy ant design.